### PR TITLE
Proxy support

### PR DESCRIPTION
**Pull request by Mike97M**
Proxy support, format of the proxy:
Proxy without authentication: proxy:port
Proxy with authentication: user:pass@proxy:port

### CODE REVIEW

File: instabot.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ class InstaBot:
     next_iteration = {"Like": 0, "Follow": 0, "Unfollow": 0, "Comments": 0}
 
     def __init__(self, login, password,
+                 proxy="",
```
Comment (LevPasha) [inline instabot.py]: Write it to the end of _init_().

File: instabot.py [hunk 2]
```diff
@@ -127,12 +128,16 @@ def __init__(self, login, password,
...
         self.log_mod = log_mod
-
         self.s = requests.Session()
         # if you need proxy make something like this:
         # self.s.proxies = {"https" : "http://proxyip:proxyport"}
         # by @ageorgios
-
+        if proxy!="":
+            proxies = {
+              'http': 'http://'+proxy,
```
Comment (LevPasha) [inline instabot.py]: Do we need http?

**Comment by LevPasha** [comment]
And plz write instruction to readme! THX!

**Comment by Mike97M** [comment]
I had no idea how to submit the edited Readme on that pull request so I've created another one: https://github.com/LevPasha/instabot.py/pull/181

**Comment by TRys8** [comment]
I edit example.py and set proxy to proxy='152.157.119.253:3128'

I get this error below.  Any suggestions?  I got the IP and port from https://free-proxy-list.net/

26.05.2018_12:21 Trying to login as heightofcolor...

Traceback (most recent call last):
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\urllib3\connectionpool.py", line 595, in urlopen
    self._prepare_proxy(conn)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\urllib3\connectionpool.py", line 816, in _prepare_proxy
    conn.connect()
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\urllib3\connection.py", line 326, in connect
    ssl_context=context)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\urllib3\util\ssl_.py", line 329, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\ssl.py", li
ne 407, in wrap_socket
    _context=self, _session=session)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\ssl.py", li
ne 814, in __init__
    self.do_handshake()
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\ssl.py", li
ne 1068, in do_handshake
    self._sslobj.do_handshake()
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\ssl.py", li
ne 689, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c
:833)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\requests\adapters.py", line 440, in send
    timeout=timeout
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\urllib3\connectionpool.py", line 639, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\urllib3\util\retry.py", line 388, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.instagram.com',
port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SS
L: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:833)'),))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "example.py", line 35, in <module>
    ["!", "!!", "!!!"]]
  File "C:\Users\TRyan\InstaBots\heightofcolor\src\instabot.py", line 232, in __
init__
    self.login()
  File "C:\Users\TRyan\InstaBots\heightofcolor\src\instabot.py", line 282, in lo
gin
    r = self.s.get(self.url)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\requests\sessions.py", line 521, in get
    return self.request('GET', url, **kwargs)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\requests\sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\requests\sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\TRyan\AppData\Local\Programs\Python\Python36-32\lib\site-packag
es\requests\adapters.py", line 506, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='www.instagram.com', port
=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: C
ERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:833)'),))

**Pull request merged**